### PR TITLE
Update Solace Cloud Foundry Spring Cloud Connector and JCSMP/JMS autoconfigure dependencies

### PR DESCRIPTION
**Pull request by Nephery**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,27 +55,27 @@ This application makes use of the Spring Cloud Connectors project to automatical
...
 
 	CloudFactory cloudFactory = new CloudFactory();
 	Cloud cloud = cloudFactory.getCloud();
-	SolaceMessagingInfo solaceMessagingServiceInfo = (SolaceMessagingInfo) cloud.getServiceInfo("MyService");
+	SolaceServiceCredentials solaceMessagingServiceInfo = (SolaceServiceCredentials) cloud.getServiceInfo("MyService");
 	
 Alternatively applications could search through the environment and discover matching services as follows:
 
-	SolaceMessagingInfo solaceMessagingServiceInfo = null;
+	SolaceServiceCredentials solaceMessagingServiceInfo = null;
```
Comment (mdspielman) [inline README.md]: This variable name doesn't make sense with the name change.

File: docs/_tutorials/spring-cloud-autoconf-java.md [hunk 2]
```diff
@@ -18,7 +20,7 @@ This tutorial is similar to the [Spring Cloud]({{ site.baseurl }}/spring-cloud)
 The goal of this tutorial is to demonstrate auto injecting a [SpringJCSMPFactory]({{ site.links-ext-github-spring-jcsmp-factory-java }}){:target="_blank"} based on  the application's Cloud Foundry Service Bindings and connect to the Solace Messaging service instance.  This tutorial will show you:
 
 1. How to Autowire a [SpringJCSMPFactory]({{ site.links-ext-github-spring-jcsmp-factory-java }}){:target="_blank"} into your application
-1. How to Autowire the [SolaceMessagingInfo]({{ site.links-ext-github-solace-messaging-info-java }}){:target="_blank"} provided by the Cloud Foundry environment using Spring Cloud Connectors.
+1. How to Autowire the [SolaceServiceCredentials]({{ site.links-ext-github-solace-messaging-info-java }}){:target="_blank"} provided by the Cloud Foundry environment using Spring Cloud Connectors.
```
Comment (mdspielman) [inline docs/_tutorials/spring-cloud-autoconf-java.md]: site.links-... name should be updated to reflect name change.

File: java-app/manifest.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 applications:
 - name: solace-sample-java-app
-  memory: 512M
+  memory: 1G
```
Comment (mdspielman) [inline java-app/manifest.yml]: This seems way too high.

Comment (PhilippeKhalife) [reply java-app/manifest.yml]: We tested on more recent CF releases with higher buildpacks and increasing the memory was only way to fix it.

File: README.md (hunk 1 above)
Comment (Nephery) [reply README.md]: Yeah, I'll change that to `solaceServiceCredentials`.

File: docs/_tutorials/spring-cloud-autoconf-java.md [hunk 4]
```diff
@@ -121,11 +123,11 @@ private SpringJCSMPFactory solaceFactory;
 // This bean is for information only, it can be used to discover more about
 // the solace service in use.
 @Autowired
-SolaceMessagingInfo solaceMessagingInfo;
+SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline docs/_tutorials/spring-cloud-autoconf-java.md]: Replacement missed here.

File: docs/_tutorials/spring-cloud-autoconf-jms.md [hunk 5]
```diff
@@ -123,11 +125,11 @@ private ConnectionFactory connectionFactory;
 // This bean is for information only, it can be used to discover more about
 // the solace service in use.
 @Autowired
-SolaceMessagingInfo solaceMessagingInfo;
+SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline docs/_tutorials/spring-cloud-autoconf-jms.md]: Same here.

File: docs/_tutorials/spring-cloud-autoconf-jndi.md [hunk 6]
```diff
@@ -124,11 +126,11 @@ private JmsTemplate jmsTemplate;
 // This bean is for information only, it can be used to discover more about
 // the solace service in use.
 @Autowired
-SolaceMessagingInfo solaceMessagingInfo;
+SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline docs/_tutorials/spring-cloud-autoconf-jndi.md]: And here.

File: spring-cloud-autoconf-java/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java [hunk 7]
```diff
@@ -67,11 +67,11 @@
 	// This bean is for information only, it can be used to discover more about
 	// the solace service in use.
 	@Autowired
-	SolaceMessagingInfo solaceMessagingInfo;
+	SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline spring-cloud-autoconf-java/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java]: This name needs an update.

File: spring-cloud-autoconf-jms/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java [hunk 8]
```diff
@@ -69,11 +69,11 @@
 	// This bean is for information only, it can be used to discover more about
 	// the solace service in use.
 	@Autowired
-	SolaceMessagingInfo solaceMessagingInfo;
+	SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline spring-cloud-autoconf-jms/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java]: And here.

File: spring-cloud-autoconf-jndi/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java [hunk 9]
```diff
@@ -68,11 +68,11 @@
 	// This bean is for information only, it can be used to discover more about
 	// the solace service in use.
 	@Autowired
-	private SolaceMessagingInfo solaceMessagingInfo;
+	private SolaceServiceCredentials solaceMessagingInfo;
```
Comment (mdspielman) [inline spring-cloud-autoconf-jndi/src/main/java/com/solace/samples/cloudfoundry/springcloud/controller/SolaceController.java]: Needs update.

File: java-app/manifest.yml (hunk 3 above)
Comment (mdspielman) [reply java-app/manifest.yml]: Seems unreasonably high. But for now, that's not a specific concern of this review.

**Pull request merged**